doc/howtos/booting-and-installation.rst

```diff
@@ -147,6 +147,126 @@ Flashing an Intel Edison requires use of a breakout board and two micro-USB cabl
+used by the kernel to describe the BeagleBone Black hardware.
+
+.. _Ostro Project download server: http://download.ostroproject.org
+
+.. comment:  next steps derived from http://www.armhf.com/boards/beaglebone-black/bbb-sd-install/
+
+#. First, verify the microSD card is using MBR partitions. Run ::
+
+   $ sudo fdisk /dev/mmcblk0
+
+   If you get an error saying "unable to open /dev/mmcblk0" then you should 
+   verify the device name assigned as described above.
+   If you get an error that GPT partitions are used, see the 
+   section below on `Converting from GPT to MBR Partitions`_ and then return to retry this step.
+   
+   If all is well, you'll see the fdisk prompt::
+
+      Command (m for help):
+
+#. We want to create two partitions on the SD card: a small primary bootable active partition, 
+   and a second primary linux root filesystem partition for the remaining space on the device.  The 
+   following ``fdisk`` commands will clean out all the existing partition information and set up two partitions:
+
+   a. Initialize the partition table by typing **o**.
+   b. Create the boot partition by typing **n** for "new", then **p** for "primary", and **1** to specify the first partition. 
+      Press enter to accept the default first sector and specify 4095 for the last sector.
+   c. Set the partition type to FAT16 by typing **t** for "type" and **e** for "W95 FAT16 (LBA)".
+   d. Set the partition active (bootable) by typing **a** then **1** (for partition 1).
+   e. Next, create the root filesystem by typing **n** for "new", then **p** for "primary", 
+      and **2** for the second partition. Accept the default values for the first and last sectors by pressing enter twice.
+   f. Type **p** to "print" the partition table. It should look about like this:: 
+
+        ...
+        Device          Boot    Start      End   Blocks     Id  System
+        /dev/mmcblk0p1    *      2048     4095     1024      e  W95 FAT16 (LBA)
+        /dev/mmcblk0p2           4096 15523839   775872     83  Linux
+
+   g. Finally, write these changes to the microSD card by typing **w** to "write" the partition table and exit.
+      
+#. At this point your microSD card is partitioned correctly but the partitions need to be formatted with
+   partition 1 as FAT and partition 2 as ext4 (the normal linux journaled filesystem)::
+
+     $ sudo mkfs.vfat  /dev/mmcblk0p1
+     $ sudo mkfs.ext4  /dev/mmcblk0p2
+
+   This last ``mkfs``  command may take a few minutes to complete, depending on the size of your SD card.  
+   You may optionally disable periodic filesystem checks on this partition with the command::
+
+     $ sudo tune2fs -c0 -i0 /dev/mmcblk0p2
+
+#. Now we can install ``MLO`` and ``u-boot.img`` in the first partition of our microSD card.  We'll start by 
+   downloading the u-boot files and then untar them to the SD card's boot partition ::
+   
+     $ wget http://s3.armhf.com/dist/bone/bone-uboot.tar.xz
```

Comment:
These instructions should read:
$ mkdir boot
$ sudo mount /dev/mmcblk0p1 boot
$ sudo cp MLO u-boot.img boot/
$ sudo umount boot/